docs/releases/5.0.txt

```diff
@@ -133,10 +133,11 @@ More options for declaring field choices
 ----------------------------------------
 
 :attr:`.Field.choices` *(for model fields)* and :attr:`.ChoiceField.choices`
-*(for form fields)* use a list of 2-tuples as the canonical form for declaring
-choices values. :ref:`field-choices-enum-types`, introduced in Django 3.0,
-required use of the ``.choices`` attribute to provide the choices in the
-expected form::
+*(for form fields)* allow for more flexibility when declaring their values. In
+previous versions of Django, ``choices`` should either be a list of 2-tuples,
+or an instance of :ref:`field-choices-enum-types`, but the latter required
```

Comment:
"... an instance of ..." isn't quite correct as an enum is not instantiated. You pass the enum class itself - an instance is an enum member.

Reply:
Thanks @ngnpope, will tweak that.

On (un)related news, I found out this branch now allows for field choices to be callables (yey) but if they are, every time `makemigrations` is run, a new migration is created, even without making any chance at all. I'll investigate more tomorrow.